isis/src/base/apps/cam2cam/cam2cam.h

```diff
@@ -26,11 +35,14 @@ namespace Isis {
       int p_inputLines;
       int p_outputSamples;
       int p_outputLines;
+      bool p_offbody;
+      bool p_trim;
 
     public:
       // constructor
       cam2camXform(const int inputSamples, const int inputLines, Isis::Camera *incam,
-                   const int outputSamples, const int outputLines, Isis::Camera *outcam);
+                   const int outputSamples, const int outputLines, Isis::Camera *outcam,
+                   const bool offbody = false, const bool trim = true);
```

Comment:
I think both offbody and trim need to be set to true here to avoid api breakage

Reply:
Not sure what you mean. Current cam2cam default behavior is to map only valid lat/lon data. It has never projected RA/DEC data. Thus, offbody=false must remain in order to preserve existing application behavior. 

I don't think this configuration breaks API because these parameters do not exist in the current API and appropriate default value are provide for the new parameters to preserve existing behavior.

Reply:
My mistake, I thought the current behavior did map off-body